app/menu.js

```diff
@@ -53,6 +53,34 @@ var createDefaultMenu = function(app, checkForUpdates) {
           ]
         },
         {
+          label: 'File',
+          submenu: [
+            {
+              label: 'New Message',
+              accelerator: 'Command+N',
+              click: function() {
+                var mainWindow = BrowserWindow.fromId(1);
+                mainWindow.webContents.send('triggerWindowEvent', 'open-mixmax-compose');
```

Comment:
Using `triggerWindowEvent`/ `onWindowEvent` isn't right for this since this isn't an event of the window—calling `onWindowEvent` will cause Electron to call `mainWindow.on('open-mixmax-compose')` which doesn't make any sense. I'll refactor this API to be clearer, and make an event-passing API for other events.